Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

EXCLUDE_PATTERNS part was reported to upstream via:
https://bz.apache.org/bugzilla/show_bug.cgi?id=59546

--- docs/doxygen.conf
+++ docs/doxygen.conf
@@ -31,8 +31,12 @@
 #STRIP_FROM_PATH=/buildpath/apr
 
 EXCLUDE_PATTERNS="*/acconfig.h" \
-                 "*/test/*" \
-                 "*/arch/*"
+                 "*/test/abts*" \
+                 "*/test/test*" \
+                 "*/include/arch/*"
 
-GENERATE_TAGFILE=docs/dox/apr.tag
+GENERATE_LATEX=NO
+HTML_TIMESTAMP=NO
+HTML_OUTPUT=manual
+SEARCHENGINE=NO
 
